py_zipkin/logging_helper.py

```diff
@@ -97,6 +101,7 @@ def log_spans(self):
                 )
 
             # Collect, annotate, and log client spans from the logging handler
+            # In case of root client span client_spans list is empty
```

Comment:
I'd be totally cool with that if you'd like to tack that on to this PR. Otherwise, I can do that as a followup task.